### DISCUSSION TITLE
What's the difference between task_work_path and work_base?

### DISCUSSION THREAD

**Discussion by tuoping** [deepmodeling/dpdispatcher#138] [Q&A]
In the example python script in README.md. There are two variables   I find confusing.
```python
task1 = Task(..., task_work_path='dir1/', ...)
submission = Submission(work_base='lammps_md_300K_5GPa/',...)
```
I don't quite understand. What exactly is the difference between `task_work_path` and `task_base`?

**Comment by njzjz** [comment]
`task_work_path` is relative to `task_base`.
